### PR TITLE
Replace underwriting with lending or loan qualification

### PR DESCRIPTION
**Pull request by mcclowes**

### CODE REVIEW

File: docs/assess/guides/loan-qualification/introduction.md [hunk 1]
```diff
@@ -24,19 +24,19 @@ This guide is for tech-savvy backend developers who know how to use an API. No f
...
 - Building any application form and dashboard UIs,
-- Details or recommendations of various financial data points used in different underwriting models.
+- Details or recommendations of various financial data points used in different lending models.
 
 ### About the demo app
 
 The app's codebase is available via our [GitHub repo](https://github.com/codatio/demo-loan-qualification). Clone it locally to try out the application.
 
-The codebase uses Codat and its [Assess](/assess/overview) product to make an underwriting decision. We will walk you through the setup in the Portal and the code [next](/assess/guides/underwriting/setting-up). 
+The codebase uses Codat and its [Assess](/assess/overview) product to make a lending decision. We will walk you through the setup in the Portal and the code [next](/assess/guides/lending/setting-up).
```
Comment (pzaichkina) [inline docs/assess/guides/loan-qualification/introduction.md:39]: so is it lending or loan qualification that should be in the URL path?

File: docs/assess/guides/loan-qualification/process-loan.md [hunk 2]
```diff
@@ -81,30 +83,30 @@ Select the **Codat Sandbox** as the source of accounting data.
 * You don't need to enter any credentials to authorize this connection. 
 * You should also skip the step of uploading business documents. 
 
-![](/img/use-cases/underwriting/sandbox-credentials-modal.png)
+![](/img/use-cases/loan qualification/sandbox-credentials-modal.png)
```
Comment (pzaichkina) [inline docs/assess/guides/loan-qualification/process-loan.md:86]: your find and replace broke our image links :)

Comment (pzaichkina) [inline docs/assess/guides/loan-qualification/process-loan.md:86]:
```suggestion
![](/img/use-cases/loan-qualification/sandbox-credentials-modal.png)
```

File: docs/assess/guides/loan-qualification/process-loan.md [hunk 3]
```diff
@@ -81,30 +83,30 @@ Select the **Codat Sandbox** as the source of accounting data.
...
+In our demo, one account remains without a specified category. You need to assign a category to it before the demo application is ready for loan qualification. This is because the Profit and Loss, and Balance Sheet data types in Assess depend on fully categorized accounts.
 
-To do that, click on the **Companies** tab in the top menu in the [Codat Portal](https://app.codat.io/). Next, click on the company you are performing underwriting for, and navigate to **Products > Assess**. Click the red **Categorization required** button to the right of the company name. 
+To do that, click on the **Companies** tab in the top menu in the [Codat Portal](https://app.codat.io/). Next, click on the company you are performing loan qualification for, and navigate to **Products > Assess**. Click the red **Categorization required** button to the right of the company name. 
 
-![](/img/use-cases/underwriting/0000-acct-categorization-modal-06-03-2023.png)
+![](/img/use-cases/loan qualification/0000-acct-categorization-modal-06-03-2023.png)
```
Comment (pzaichkina) [inline docs/assess/guides/loan-qualification/process-loan.md:98]:
```suggestion
![](/img/use-cases/loan-qualification/0000-acct-categorization-modal-06-03-2023.png)
```

File: docs/assess/guides/loan-qualification/process-loan.md [hunk 4]
```diff
@@ -81,30 +83,30 @@ Select the **Codat Sandbox** as the source of accounting data.
...
 
 This takes you to the **Account categorization** page which displays the uncategorized account. **Account type** and **Account subtype** are pre-filled for you. **Categories version** should be set to _Version 2_. Select _Accounts Payable_ as **Account detail** in the drop down and **Save** the categorization. 
 
-![](/img/use-cases/underwriting/0000-categorization-screen.png)
+![](/img/use-cases/loan qualification/0000-categorization-screen.png)
```
Comment (pzaichkina) [inline docs/assess/guides/loan-qualification/process-loan.md:102]:
```suggestion
![](/img/use-cases/loan-qualification/0000-categorization-screen.png)
```

File: docs/assess/guides/loan-qualification/process-loan.md [hunk 5]
```diff
@@ -114,18 +116,18 @@ Try these suggestions to make the most of your experience with the demo app:
...
 - **Underwrite using different datasets**  
   Start another loan application, and choose a different Sandbox company type to get a different set of financial data to be used in the decision-making. 
 
 - **Use a company's real data**  
   Take the demo one step further and use real credentials to access existing financial data in an accounting platform. Set up the [integration](/integrations/accounting/overview) you plan to use, and connect to it while following the auth flow. Then, review how the app makes a decision based on your company's real data. 
 
-- **Inspect the underwriting logic**  
-  We provide [detailed information](/assess/guides/underwriting/uw-decision) about the underwriting logic we included in our demo app, and how exactly the financial data is fetched. 
+- **Inspect the loan qualification logic**  
+  We provide [detailed information](/assess/guides/loan qualification/uw-decision) about the loan qualification logic we included in our demo app, and how exactly the financial data is fetched.
```
Comment (pzaichkina) [inline docs/assess/guides/loan-qualification/process-loan.md:128]:
```suggestion
  We provide [detailed information](/assess/guides/loan-qualification/uw-decision) about the loan qualification logic we included in our demo app, and how exactly the financial data is fetched. 
```

**Review by pzaichkina — CHANGES_REQUESTED** [review]

**Comment by mcclowes** [comment]
Apologies @pzaichkina , very messy on my part

**Review by d-laing — APPROVED** [review]
All looks good.

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**